config/openiotsdk/cmake/sdk.cmake

```diff
@@ -37,11 +37,12 @@ endif()
 
 # Overwrite versions of Open IoT SDK components
 
-# Add a Matter specific version of Mbedtls
+# Fetch the same version of mbedtls as TF-M
+# You can find the TF-M settings in TFM_SOURCE_DIR/config/config_base.cmake
 FetchContent_Declare(
     mbedtls
     GIT_REPOSITORY https://github.com/ARMmbed/mbedtls
-    GIT_TAG        v3.2.1
+    GIT_TAG        mbedtls-3.2.1
```

Comment:
We can prevent multiple download by providing the path to TFM: https://tf-m-user-guide.trustedfirmware.org/building/tfm_build_instruction.html?highlight=mbedcrypto_version#prepare-mbed-crypto-repository

Reply:
What path do you mean?